packages-ts/eqlabs-multisig-integration/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
No need to include `./test`, we can use just our standard config here: https://github.com/smartcontractkit/chainlink-starknet/blob/develop/packages-ts/starknet-gauntlet/tsconfig.json (c/p from here)

Reply:
We don't have `src` here or it will be added in the future (?) + we need to add path to the apt hardhat.config.ts

Reply:
This is just an internal integration package, will not be published ever and probably no src will be added, just tests.

Not sure if there is some problem here with my original suggestion, but it’s a minor thing so feel free to keep it as is so we get this merged.